src/components/AssetsEditor/SingleAssetEditor/SingleAssetEditor.tsx

```diff
@@ -67,8 +66,9 @@ export default class SingleAssetEditor<T extends RawAsset | Asset> extends React
     const { asset } = this.props
 
     let mappings: Asset['contents'] | RawAsset['contents']
+    const contentValues = Object.values(asset.contents)
 
-    if (!isRemoteURL(asset.thumbnail)) {
+    if (typeof contentValues[0] !== 'string') {
       mappings = rawMappingsToObjectURL((asset as RawAsset).contents)
     } else {
       mappings = Object.keys((asset as Asset).contents).reduce<Asset['contents']>((acc, path) => {
```

Comment:
I know it's not related to this PR, but there's a `as any` below this line that I think it's not necessary